Tests/Tests.csproj

```diff
@@ -12,6 +13,8 @@
     <FileAlignment>512</FileAlignment>
     <OutputPath>..\_build\out\$(AssemblyName)\$(Configuration)\bin\</OutputPath>
     <IntermediateOutputPath>..\_build\out\$(AssemblyName)\$(Configuration)\obj\</IntermediateOutputPath>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
```

Comment:
Are these lines needed? They don't look like they would do much since there's no content between them.

Reply:
Yeah, don't know, Visual Studio inserted them itself.
I'll try to delete them.